mariadb.spec

```diff
@@ -1307,6 +1349,52 @@ fi
 %endif
 
 %changelog
+* Tue Jan 10 2017 Michael Bayer <mbayer@redhat.com> - 3:10.1.20-1
+- Backport Fedora changes since Nov 16
+- Update to version 10.1.20
+
+* Tue Jan 03 2017 Honza Horak <hhorak@redhat.com> - 3:10.1.20-3
```

Comment:
do we care that the changelog has these bogus (to RDO) release numbers in it?  can we mark "(fedora)" or something or it doesnt matter?